samples/TypeScript/tsxAttributeResolution9.tsx

```diff
@@ -1,27 +0,0 @@
-//@jsx: preserve
```

Comment:
This file is taken from a test suite that is testing features of the compiler (to pass-through JSX), it is not a file that’s representative of a file in a “real application” imo.